test/cbmc/proofs/ARP/ARPGenerateRequestPacket/ARPGenerateRequestPacket_harness.c

```diff
@@ -24,9 +28,15 @@ void harness()
     NetworkBufferDescriptor_t xNetworkBuffer2;
 
     xNetworkBuffer2.pucEthernetBuffer = xBuffer;
```

Comment:
Would it be more elegant to write 
```
xNetworkBuffer2.xDataLength = ucBUFFER_SIZE;
xNetworkBuffer2.pucEthernetBuffer = malloc(xNetworkBuffer2.xDataLength);
__CPROVER_assume( xNetworkBuffer2.pucEthernetBuffer != NULL );
```
I'm not sure what the answer is.

Reply:
Sure it would be easy to read. I have updated the code.